cdisc-org/cdisc-rules-engine
# Capitalize USUBJID and SEQ in results, update tests

Steps to test:

1. Run unit tests
2. Run a validation and ensure usubjid and seq are populated in the report
